src/client/results/mostfrequent/MostFrequent.js

```diff
@@ -1,9 +1,19 @@
 import React, { Component } from 'react';
 import SlidingSearchBar from './SearchBar';
 
-const MostFrequent = ({ title, filterTerm, onTypingInSearchBar, onToggleAll, currentToggledAction, children }) => (
-  <div>
-    <h3>{title}</h3>
+const MostFrequent = ({
```

Comment:
I'm not going to ask you to refactor, but this issue definitely correct - 8 parameters? Some of them could (LATER not today) be bunched into a singular object potentially